src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+@import './variables';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-cell-gap;
+  width: $row-width;
+  padding: 10px;
+
+  &__day {
+    position: relative;
+    display: none;
+    box-sizing: border-box;
+    min-width: $day-cell-size;
+    height: $day-cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+
+    transition: all 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: $day-hover-color;
+      cursor: pointer;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        font-size: 30px;
```

Comment:
- All of them have the same styles, so apply them only once instead of doing it in the loop.
- And it's not a good practice to center with absolute position, you can just center with flex for the parent container